core/src/main/java/com/sequenceiq/cloudbreak/core/flow2/chain/UpgradeDatalakeFlowEventChainFactory.java

```diff
@@ -0,0 +1,61 @@
+package com.sequenceiq.cloudbreak.core.flow2.chain;
+
+
+import static com.sequenceiq.cloudbreak.core.flow2.cluster.upgrade.ClusterUpgradeEvent.CLUSTER_UPGRADE_EVENT;
```

Comment:
Are you sure this is the right class? There is a new one below with the same name added to package `com.sequenceiq.cloudbreak.core.flow2.cluster.datalake.upgrade`. I also wonder if you meant to trigger `ClusterUpgradeFlowConfig` whose init state is `CLUSTER_MANAGER_UPGRADE_EVENT` (from `c.s.c.c.f.c.datalake.upgrade.ClusterUpgradeEvent`).